dev-tools/phpstan/baseline.php

```diff
@@ -1555,18 +1555,6 @@
 	'count' => 1,
 	'path' => __DIR__ . '/../../src/Fixer/Operator/BinaryOperatorSpacesFixer.php',
 ];
-$ignoreErrors[] = [
-	'message' => '#^Method PhpCsFixer\\\\Fixer\\\\Operator\\\\TernaryToElvisOperatorFixer\\:\\:getAfterOperator\\(\\) should return array\\{start\\: int, end\\: int\\} but returns array\\{start\\: int\\|null, end\\?\\: int\\|null\\}\\.$#',
-	'identifier' => 'return.type',
-	'count' => 1,
-	'path' => __DIR__ . '/../../src/Fixer/Operator/TernaryToElvisOperatorFixer.php',
-];
-$ignoreErrors[] = [
-	'message' => '#^Offset 1\\|2\\|3\\|4\\|5\\|6\\|7\\|8\\|9\\|10\\|11\\|12\\|13\\|14 might not exist on array\\<1\\|2\\|3\\|4\\|5\\|6\\|7\\|8\\|9\\|10\\|11\\|12\\|13\\|14, array\\{start\\: array\\{int, string\\}\\|string, end\\: array\\{int, string\\}\\|string\\}\\>\\.$#',
```

Comment:
It looks more like an issue in PHPStan for me - `Offset of GIVEN_TYPE might not exist on array<GIVEN_TYPE, NON_EMPTY_ARRAY>` cannot happen, right?

I cannot reproduce it on https://phpstan.org, because we have `reportPossiblyNonexistentGeneralArrayOffset: true` and it is not enabled there (I copied the whole `Tokens::getBlockEdgeDefinitions` and still did not get it).

Reply:
it can happen. index is of type GIVEN_TYPE, but it doesn't mean all options are used. think about it like array<int, string>. it doesn't mean each int exists in array

Reply:
> it doesn't mean all options are used

according to https://phpstan.org/writing-php-code/phpdoc-types it does:

> `self::SOME_*` (all constants on `self` that start with `SOME_`)

Reply:
it says that all constants are allowed and only they are allowed, but it does not say they are constants are ensured to be present

https://phpstan.org/r/3b08d121-74ac-4cdc-9dc3-bfb7ecfd030b

Reply:
(the same as with `int` analogy - all ints are allowed, but not all ensured to be present)